custom_components/mail_and_packages/const.py

```diff
@@ -12,7 +12,7 @@
 
 DOMAIN = "mail_and_packages"
 DOMAIN_DATA = f"{DOMAIN}_data"
-VERSION = "0.0.0-dev"  # Now updated by release workflow
+VERSION = "0.3.14"
```

Comment:
```suggestion
VERSION = "0.0.0-dev"  # Now updated by release workflow
```
We don't update this manually.